frontend/src/assets/css/journey/comparison.css

```diff
@@ -0,0 +1,46 @@
+.comparisonP {
+  /* margin: auto auto; */
+}
+
+.compContent {
+  min-height: 60vh;
+  text-align: right;
+  width: 100%;
+}
+
+.compContentContent {
+  margin-top: 7em;
+  text-align: right;
+  /* max-width: 20em; */
+  /* padding-right: 2em; */
+}
+
+.compContentContent h1 {
+  /* padding-right:  */
```

Comment:
What's with this double naming?
`compContentContent`
`compVisVis`

Reply:
div in div

Reply:
It's still bad :')

Reply:
i'll change soon.